invenio/modules/knowledge/api.py

```diff
@@ -321,15 +334,15 @@ def add_kb(kb_name="Untitled", kb_type=None, tries=10):
             name = kb_name + " " + str(index)
             i = i + 1
             created = False
-        except:
+        except Exception:
             db.session.rollback()
             raise
 
     if created is False:
         # TODO raise the right exception
-        raise Exception("Can't create kb \"{0}\".\n" +
-                        "Probabily the server is busy! " +
-                        "Try again later.".format(kb_name))
+        raise Exception(_("Can't create kb \"{0}\".\n"
```

Comment:
anyway if you would like to make it translatable you need to use following format:
```python
_("Message %(x_variable_name)s bar.", x_variable_name="John Doe")
```